gradle.properties

```diff
@@ -1,3 +1,4 @@
-version=0.0.2-SNAPSHOTorg.gradle.daemon   = true
+version=0.0.2-SNAPSHOT
```

Comment:
Can you please put this is a separate PR?

Reply:
Oh yes, sure.